### PR TITLE
Allow setting the opsctl user when generating

### PR DESCRIPTION
**Pull request by MarcelMue**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: cmd/generate/vault.go [hunk 1]
```diff
@@ -38,9 +38,14 @@ func newVaultClient(config vaultClientConfig) (*vaultapi.Client, error) {
 	return vaultClient, nil
 }
 
-func createVaultClientUsingOpsctl(ctx context.Context, gitHubToken, installation string) (*vaultapi.Client, error) {
+func createVaultClientUsingOpsctl(ctx context.Context, gitHubToken, user, installation string) (*vaultapi.Client, error) {
 	cmdArgs := []string{"opsctl", "create", "vaultconfig", "-i", installation, "-o", "json"}
 
+	if user != "" {
```
Comment (MarcelMue) [inline cmd/generate/vault.go:44]: Me being lazy right now, there is probably a nicer way to append here.

**Review by axbarsan — APPROVED** [review]
LGTM

File: cmd/generate/vault.go (hunk 1 above)
Comment (kopiczko) [reply cmd/generate/vault.go:44]: I think you can do `cmdArgs = append(cmdArgs, "--user", user)`.

File: cmd/generate/flag.go [hunk 2]
```diff
@@ -16,6 +16,7 @@ const (
 	flagName          = "name"
 	flagNamespace     = "namespace"
 	flagRaw           = "raw"
+	flagUser          = "user"
```
Comment (kopiczko) [inline cmd/generate/flag.go:19]: Maybe the name could be more expressive. Like `ssh-user` or `jumphost-user`.

**Review by kopiczko — APPROVED** [review]

**Pull request merged**